synapse/handlers/e2e_keys.py

```diff
@@ -330,7 +330,8 @@ def _upload_one_time_keys_for_user(self, user_id, device_id, time_now,
                         (algorithm, key_id, ex_json, key)
                     )
             else:
-                new_keys.append((algorithm, key_id, encode_canonical_json(key)))
+                new_keys.append((
+                    algorithm, key_id, encode_canonical_json(key).decode('ascii')))
```

Comment:
Nit picky, but this is quite unreadable to me at the end, can you stick the trailing `))` on a newline?

Reply:
can we save that for running black on things? :)

Reply:
Sure, so long as we do do it